Formula/s/sevenzip.rb

```diff
@@ -38,7 +31,7 @@ def install
         ["gcc", "g"]
       end
 
-      system "make", "-f", "../../cmpl_#{mk_suffix}.mak", "DISABLE_RAR_COMPRESS=1"
+      system "make", "-f", "../../cmpl_#{mk_suffix}.mak"
```

Comment:
I'm pretty sure we want to keep this, since rar compression is not open source.

Reply:
Fair ennough.  I can change the PR accordingly.